config/db.js

```diff
@@ -0,0 +1,11 @@
+import { Sequelize } from 'sequelize';
+
+export const sequelize = new Sequelize({
```

Comment:
Move all configuration to .env file

Reply:
Ok. I try to resolve this comment on next task iteration.